src/vcVoxelShaders.h

```diff
@@ -22,8 +22,10 @@ struct vcUDRSData
     } classification;
     struct
     {
-      float minThreshold;
-      float maxThreshold;
+      float minBound;
+      float maxBound;
+      uint32_t errorColour;
+      uint32_t outOfBoundsColour;
```

Comment:
This will likely conflict with @bwockner's work

Reply:
I see there a lot of clashing with this [PR](https://github.com/Euclideon/vaultclient/pull/410). We have been working in the same area of the codebase.

Reply:
Mine is fairly small, I can fix it up if this one is approved first